src/main/java/mat/model/clause/CQLLibrary.java

```diff
@@ -332,6 +332,11 @@ public boolean isFhirMeasure() {
         return ModelTypeHelper.isFhir(getLibraryModelType());
     }
 
+    @Transient
+    public boolean isFhirLibrary(String modelType) {
+        return ModelTypeHelper.isFhir(modelType);
+    }
+
     @Transient
     public boolean isQdmMeasure() {
         return ModelTypeHelper.isQdm(getLibraryModelType());
```

Comment:
probably we should rename isQdmMeasure and isFhirMeasure methods to isQdmLibrary and isFhirLibrary

Reply:
Those 2 methods are not created now. and isFhirMeasure method is being used in CqlLibraryService class